daemon/kmd/wallet/driver/ledger.go

```diff
@@ -209,7 +209,25 @@ func (lw *LedgerWallet) SignTransaction(tx transactions.Transaction, pw []byte)
 
 // MultisigSignTransaction implements the Wallet interface.
 func (lw *LedgerWallet) MultisigSignTransaction(tx transactions.Transaction, pk crypto.PublicKey, partial crypto.MultisigSig, pw []byte) (crypto.MultisigSig, error) {
-	return crypto.MultisigSig{}, errNotSupported
+
```

Comment:
Minor nit: most (all?) of the existing code doesn't have a blank line at the beginning of the function body.